documentstore_migracao/utils/xml.py

```diff
@@ -86,9 +116,10 @@ def loadToXML(file):
 
     parser = etree.XMLParser(remove_blank_text=True, no_network=True)
     try:
-        xml = etree.parse(file, parser)
-    except etree.XMLSyntaxError as exc:
-        raise Exception(str(exc)) from None
+        content = get_fixed_xml_content(file)
+        xml = etree.parse(BytesIO(content.encode("utf-8")), parser)
+    except (etree.XMLSyntaxError, GetFixedXMLContentError) as exc:
+        raise LoadToXMLError(exc)
```

Comment:
@robertatakenaka, uma dúvida: Ao fazermos `LoadToXMLError(exc)` a exceção `exc` é serializada para _string_?

Pergunto isso porque se a exceção não for serializavel nós não conseguiremos ler o motivo dela ter sido lançada quando executarmos a conversão por exemplo (por causa do process pool worker).

Reply:
hum... não sei responder... como seria _serializada para string_?

Reply:
Antes:
```python
        raise Exception(str(exc)) from None
```
Depois
```python
raise LoadToXMLError(exc)
```
Vc diz que deveria ser:
```python
raise LoadToXMLError(str(exc)) from None
```
?

Reply:
@robertatakenaka, sim mas apenas se esse comportamento já não seja o padrão no código que você subiu.

Reply:
@joffilyfe por gentileza, pode revisar novamente?